src/ui/views/NewAccount.vue

```diff
@@ -434,29 +447,29 @@
               :persistent-hint="true"
               dense
               class="mt-0 pt-0 mb-4" />
+          </v-form>
+          <div class="form-buttons">
+            <v-btn @click="currentStep--">
+              {{ t('LabelBack') }}
+            </v-btn>
+            <v-btn
+              color="primary"
+              @click="onCreate()">
+              {{ t('LabelContinue') }}
+            </v-btn>
+          </div>
+        </v-stepper-content>
 
-            <div class="d-flex flex-row justify-space-between">
-              <v-btn @click="currentStep--">
-                {{ t('LabelBack') }}
-              </v-btn>
-              <v-btn
-                color="primary"
-                @click="onCreate()">
-                {{ t('LabelContinue') }}
-              </v-btn>
-            </div>
-          </v-stepper-content>
-          <v-stepper-content step="5">
-            <div class="headline">
-              {{ t('LabelAccountcreated') }} <v-icon>mdi-check</v-icon>
-            </div>
-            <div v-if="isBrowser">
-              {{ t('DescriptionAccountcreated') }}
-            </div>
-          </v-stepper-content>
-        </v-stepper-items>
-      </v-stepper>
-    </v-card>
+        <v-stepper-content step="5">
+          <div class="headline">
+            {{ t('LabelAccountcreated') }} <v-icon>mdi-check</v-icon>
+          </div>
+          <div v-if="isBrowser">
+            {{ t('DescriptionAccountcreated') }}
+          </div>
+        </v-stepper-content>
+      </v-stepper-items>
+    </v-stepper>
```

Comment:
<img width="651" height="233" alt="Image" src="https://github.com/user-attachments/assets/43ce52a1-8149-460f-bf46-32284ffdcf07" />

I think `DescriptionAccountcreated` needs an additional margin now

Reply:
Fixed it myself :)